client-lib/go/client/jvs_client.go

```diff
@@ -50,12 +59,50 @@ func NewJVSClient(ctx context.Context, config *JVSConfig) (*JVSClient, error) {
 	}
 
 	return &JVSClient{
-		config: config,
-		keys:   cached,
+		config:           config,
+		keys:             cached,
+		forbidBreakglass: config.ForbidBreakglass,
 	}, nil
 }
 
 // ValidateJWT takes a jwt string, converts it to a JWT, and validates the signature.
 func (j *JVSClient) ValidateJWT(jwtStr string) (*jwt.Token, error) {
+	// Handle Break-glass tokens.
+	if strings.HasSuffix(jwtStr, UnsignedPostfix) {
+		token, err := jwt.Parse([]byte(jwtStr), jwt.WithVerify(false))
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse jwt %s: %w", jwtStr, err)
+		}
+		valid, err := j.unsignedTokenValidAndAllowed(token)
+		if !valid {
+			return nil, fmt.Errorf("token unsigned and could not be validated: %w", err)
+		}
+		return &token, nil
+	}
 	return jvscrypto.ValidateJWT(j.keys, jwtStr)
 }
+
+func (j *JVSClient) unsignedTokenValidAndAllowed(token jwt.Token) (bool, error) {
+	if j.forbidBreakglass {
+		return false, fmt.Errorf("break glass tokens is forbidden, denying")
+	}
+	justs, ok := token.Get("justs")
+	if !ok {
+		return false, fmt.Errorf("can't find 'justs' in claims, denying")
+	}
+	justsBytes, err := json.Marshal(justs)
+	if err != nil {
+		return false, fmt.Errorf("failed to marshal 'justs', denying")
+	}
+	var justifications []*jvspb.Justification
+	err = json.Unmarshal(justsBytes, &justifications)
+	if err != nil {
+		return false, fmt.Errorf("failed to unmarshal 'justs', denying")
+	}
+	for _, justification := range justifications {
+		if justification.GetCategory() == BreakglassCategory {
+			return true, nil
+		}
+	}
+	return false, fmt.Errorf("unable to find correct break-glass category, denying")
```

Comment:
```suggestion
	return false, fmt.Errorf("justification category is not breakglass, denying")
```

Reply:
done. I also updated the message in java client to make it consistent.